src/fuzzing/src/test/scala/Fuzzing.scala

```diff
@@ -45,7 +45,15 @@ class Fuzzing extends TestBase {
   // TODO verify that model UIDs match the class names, perhaps use a Trait
 
   test("Verify all estimators can be turned into pipelines, saved and loaded") {
-    estimators.foreach(est => {
+    val exemptions: Set[String] = Set(
```

Comment:
Scalastyle complains because tests are macro expanded to something like classes and then public stuff needs to have type annotations